### PR TITLE
Change agent prompts from json to yaml

### PR DESCRIPTION
**Pull request by mattlindsey**
Tested both agents against openai, google palm, and postgres database.
Completes #155

### CODE REVIEW

**Comment by mattlindsey** [comment]
Whoops. Sorry. Just have to update specs too.

**Comment by mattlindsey** [comment]
@andreibondarev Fixed and tested. Could you review when you get a chance.

File: lib/langchain/agent/chain_of_thought_agent/chain_of_thought_agent_prompt.json [hunk 1]
```diff
@@ -1,10 +0,0 @@
-{
-  "_type": "prompt",
-  "template": "Today is {date} and you can use tools to get new information. Answer the following questions as best you can using the following tools:\n\n{tools}\n\nUse the following format:\n\nQuestion: the input question you must answer\nThought: you should always think about what to do\nAction: the action to take, should be one of {tool_names}\nAction Input: the input to the action\nObservation: the result of the action\n... (this Thought/Action/Action Input/Observation can repeat N times)\nThought: I now know the final answer\nFinal Answer: the final answer to the original input question\n\nBegin!\n\nQuestion: {question}\nThought:",
```
Comment (andreibondarev) [inline lib/langchain/agent/chain_of_thought_agent/chain_of_thought_agent_prompt.json:3]: Just curious why'd we remote the final "Thought:"?

Comment (mattlindsey) [reply lib/langchain/agent/chain_of_thought_agent/chain_of_thought_agent_prompt.json:3]: I thought that demonstrates to the ai how to format the final thought as "Final Thought:", but I can try taking that out.

Comment (andreibondarev) [reply lib/langchain/agent/chain_of_thought_agent/chain_of_thought_agent_prompt.json:3]: Yeah, the scope of the ticket was to just convert the prompt from JSON to YAML. 

Thanks for being flexible!

Comment (mattlindsey) [reply lib/langchain/agent/chain_of_thought_agent/chain_of_thought_agent_prompt.json:3]: @andreibondarev Sure, but note that I don't think I changed the prompt for ChainOfThought.  I just moved it to yaml.

But I did just cleanup SQLQueryAgent prompting slightly and tested it.

**Comment by mattlindsey** [comment]
@andreibondarev I think this is good now.

**Pull request closed without merging**